cmd/main.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/Sirupsen/logrus"
+	config "github.com/openebs/zfs-localpv/pkg/config/v1alpha1"
```

Comment:
Let us avoid v1alpha1 folders in business logic packages.
We can have them only in the apis i.e. schema packages.

Reply:
make sense.